docs/en/delayed.md

```diff
@@ -1,21 +1,50 @@
 # `delayed()`
 
-Same as `lazy` but postpones component loading for even longer. Delay, JavaScript idle time and [DRAF](https://github.com/ryanve/draf) can be specified.
+Works similar to [`lazy()`](./lazy.md) method but it can postpones loading or rendering of the component.
+
+Delaying rendering is useful to prevent [FOUC](https://en.wikipedia.org/wiki/Flash_of_unstyled_content) in some cases, for example, if you
+are using CSS modules with Webpack.
+
+Delaying loading is useful if your component is not very important and you want to use CPU resources for other things or start loading the
+component only on [browser idle time](https://developer.mozilla.org/en-US/docs/Web/API/Window/requestIdleCallback).
 
 Using default params with this function is equivalent to using [`lazy()`](./lazy.md) function.
 
-## Reference
 
-```ts
-type TDelayed = <TProps>(params: IDelayedParams<TProps>) => ILazyComponent<TProps>;
+## Examples
+
+Lazy load a component and prevent [FOUC](https://en.wikipedia.org/wiki/Flash_of_unstyled_content).
 
-interface IDelayedParams<TProps> extends ILazyParams<TProps> {
-  delay?: number;
-  draf?: boolean;
-  idle?: boolean;
-}
+```jsx
+const loader = () => import('./MyComponent'); // Assume it is exports as default exports.
```

Comment:
Typo: `// Assume it exports as default export.`

Reply:
done